lib/resin-lint.ts

```diff
@@ -1,10 +1,12 @@
 import * as Bluebird from 'bluebird';
+// tslint:disable-next-line:no-var-requires
 const coffeelint: any = require('coffeelint');
+// tslint:disable-next-line:no-var-requires
 const reporter: any = require('coffeelint/lib/reporters/default');
 import * as depcheck from 'depcheck';
 import * as fs from 'fs';
 import * as glob from 'glob';
-const merge: any = require('merge');
+import { merge } from 'lodash';
```

Comment:
I switched to using lodash merge because it has typings and I know it's maintained